lib/wallaroo/core/source/connector_source/connector_source_listener.pony

```diff
@@ -323,3 +413,47 @@ actor ConnectorSourceListener[In: Any val] is SourceListener
       @pony_os_socket_close[None](_fd)
       _fd = -1
     end
+
+  // Application startup lifecycle events
```

Comment:
This looks right.  Elsewhere in the code you've left remnants of the old way of managing starting listening and sources.  Is that just transitional code that you're removing?